### DISCUSSION TITLE
No user account or password for administrator

### DISCUSSION THREAD

**Discussion by jayanh** [SafeExamBrowser/seb-win-refactoring#167] [General]
Hi, I just setup SEB server v1.2. 

As the document said after SEB Server successfully started the account and password will be collected from log by command "docker logs ${container name}". But in my log i did not see any where for account to login.

Anyone know how to get administrator account?

Thanks

Jayanh

**Comment by anhefti** [comment]
Hi Jayanh 

Can you please send us more information about your SEB Server setup. Did you setup the productive version or the demo version. The initial account is only generated within the productive setup and also only on the first initialization of the database. There will be no account output on further restarts of the container once the database has been initialized. You can try the following:

1. Check the database "user" table if there is one entry. If so, the initial account has successfully been created by the SEB Server. Check later logs or log-files to find the output you need, it should be there.
2. If this doesn't work, try to drop the SEB Server database (make a backup if you want but there should not be any needed data since you where not able to login). Then restart the SEB Server docker container again, with docker-compose down, docker-compose up. The SEB Server should initialize the database and the account again put the data to the output log -> docker logs seb-server

**Reply by jayanh** [reply]
Hi Anhefti,

Thanks for your help. I did drop system and reinstall and found the user and password as your second steps.

Thanks and regards

Jayanh
